chains/schema.json

```diff
@@ -285,6 +285,10 @@
           "type": "string",
           "enum": [
             "arbitrumnitro",
+            "opstack",
+            "polygoncdk",
+            "polkadotsubstrate",
+            "zksync",
```

Comment:
Wouldn't it be a good idea to add a value also for cosmos chains?

Reply:
we already have the `ProtocolType` enum which kinda covers that already, same with solana/sealevel